docs/reference/operations/lookup_accounts.md

```diff
@@ -10,3 +10,18 @@ An [`id`](../accounts.md#id) belonging to a [`Account`](../accounts.md).
 
 - If the account exists, return the [`Account`](../accounts.md).
 - If the account does not exist, return nothing.
+
+## Client libraries
+
+For language-specific docs see:
+
+* [Looking up accounts using the Java library](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients/java#creating-accounts)
+* [Looking up accounts using the Go library](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients/go#creating-accounts)
+* [Looking up accounts using the Node.js library](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients/node#creating-accounts)
```

Comment:
Is "creating accounts" the right header to link to here?

Reply:
Good catch